developer/doc/ReleaseNotes/ReleaseNotes_template.md

```diff
@@ -78,16 +80,33 @@ You can also refer to the [OpenStudio SDK Python Binding Version Compatibility M
 
 ## Minor changes and bug fixes
 
+Refer to the changelog on the release page at [v@VERSION@](https://github.com/NREL/OpenStudio/releases/v@VERSION@)
 
 **Full Changelog**: https://github.com/NREL/OpenStudio/compare/v@VERSION@...v@VERSION@
 
 **New Contributors**:
 
-### OpenStudio Standards vXXX
+### OpenStudio Standards v@STANDARDS_VERSION@
+
+Update the openstudio-standards gem to version [@STANDARDS_VERSION@](https://github.com/NREL/openstudio-standards/releases/tag/v@STANDARDS_VERSION@)
+In addition to some refactoring, this release also included conversion of 90.1 data to formal database.
 
 
 ### OpenStudio Server vXXX
 
 
-## Issue Statistics Since Previous Release
 
+
+---
+# This YAML header controls the pandoc (via TeX) to PDF settings
+# To convert the markdown to pdf, do `pandoc release_notes.md -o release_notes.pdf`
+title:  'OpenStudio Release Notes - @VERSION@'
+author:
+- National Renewable Energy Laboratory
+colorlinks: true
+linkcolor: blue
+urlcolor: Mahogany
+toccolor: gray
+geometry:
+- margin=1in
+---
```

Comment:
I also adjusted the Release_Notes template.md to conform to latest format, and added some quick `pandoc` settings to set the title, margin and colors. And most importantly, a reminder of how to generate the PDF from the Markdown file.